.github/workflows/client-build-en.yaml

```diff
@@ -0,0 +1,47 @@
+name: Client Build with EN
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - 'api/**'
+      - 'client/exposure_notifications/**'
+      - 'client/app/**'
+      - 'client/app_en/**'
+      - 'client/flutter/**'
```

Comment:
I'm not sure we want to run this CI for changes to the non-EN-flavored app... building the client once already takes 8-10 minutes or more, this will slow down our CI substantially

Reply:
Let me run a test.  A primary purpose for this change is that neither v1 nor EN break each other.  Let me see what the total difference in time is.

Reply:
They do run in parallel but it does affect our github repo budget.  Will remove the app builds for now.

Reply:
What I did is make the build run only on a pubspec change.